packages/wallet/frontend/src/pages/grant-interactions/index.tsx

```diff
@@ -49,9 +51,14 @@ const GrantInteractionPage = ({
     openDialog(
       <SuccessDialog
         onClose={closeDialog}
+        onSuccess={() => {
+          router.push(
+            `${process.env.NEXT_PUBLIC_AUTH_HOST}/interact/${interactionId}/${nonce}/finish`
+          )
+          closeDialog()
+        }}
```

Comment:
Why don't we automatically redirect the user after a few seconds instead of letting them click the `Finish` button?

Reply:
There is the finish api call from interaction with grants, we want to emphasise that.
It is definetly better to have this one button then also Close :D

Reply:
I think the `finish` action is just an implementation detail. The user should *always* be redirected after interacting with a grant.